src/engine/strat_engine/backstore/devices.rs

```diff
@@ -358,70 +501,146 @@ pub fn wipe_blockdevs(blockdevs: &[StratBlockDev]) -> StratisResult<()> {
 
 #[cfg(test)]
 mod tests {
-    use std::fs::OpenOptions;
+    use std::{error::Error, fs::OpenOptions};
 
     use uuid::Uuid;
 
-    use crate::engine::strat_engine::{
-        backstore::{
-            identify::find_all_block_devices_with_stratis_signatures, metadata::device_identifiers,
-            setup::get_metadata,
+    use crate::engine::{
+        engine::BlockDev,
+        strat_engine::{
+            backstore::{
+                crypt::CryptHandle, identify::find_all_block_devices_with_stratis_signatures,
+                metadata::device_identifiers, setup::get_metadata,
+            },
+            cmd,
+            tests::{crypt, loopbacked, real},
         },
-        cmd,
-        tests::{loopbacked, real},
     };
 
     use super::*;
 
     /// Test that initialing devices claims all and that destroying
     /// them releases all.
-    fn test_ownership(paths: &[&Path]) {
+    fn test_ownership(
```

Comment:
While you're here could you fix the spelling mistake: ```initialing``` -> ```initializing```.

Reply:
Resolved.